### PR TITLE
An example pull request

### PR DESCRIPTION
**Pull request by Edgar-Saavedra**
@soniapargas Hey check out the new functions let me know if in need anything else.

### CODE REVIEW

File: test/example.py [hunk 1]
```diff
@@ -0,0 +1,8 @@
+def a_new_function():
+  return "whats up"
+
+def second_function():
+  return "\nwhats up again"
+
```
Comment (soniapargas) [inline test/example.py:6]: @Edgar-Saavedra We need to add a third function that returns "busy."

Comment (Edgar-Saavedra) [reply test/example.py:6]: @soniapargas yeah lets call it 3rd_function and add it here.

Comment (Edgar-Saavedra) [reply test/example.py:6]: @soniapargas can you add it?

Comment (soniapargas) [reply test/example.py:6]: @Edgar-Saavedra thanks. I had to rename the function to "third_function"

**Pull request merged**